install/docker/debian-docker-install.md

```diff
@@ -6,16 +6,27 @@
 >   [already installed the Kata Containers packages](../debian-installation-guide.md).
 > - this guide allows for installation with `systemd` or `sysVinit` init systems
 
-1. Install the latest version of Docker with the following commands:
-
-   > **Note:** This step is only required if Docker is not installed on the system.
+1. Install Docker with the following commands:
+
+   > **Notes:**
+   >
+   > - This step is only required if Docker is not installed on the system.
+   > - Newer versions of Docker have
+   >   [removed devicemapper support](https://github.com/kata-containers/documentation/issues/373)
+   >   so the commands below install the latest version which includes
```

Comment:
Suggested rewrite:
so the following commands install the latest version, which includes